editor/src/clj/editor/engine.clj

```diff
@@ -174,6 +174,13 @@
              {:log-port log-port
               :address loopback-address}))))
 
+;; Parse a line from engine output to extract engine version info.
+(defn parse-engine-version-line [output]
+  (some (fn [line]
+          (when-let [[_ version sha] (re-find #"INFO:ENGINE: Defold Engine ([^\s]+) \(([^)]+)\)" line)]
+            line))
```

Comment:
Since you always pass a single line here, there is no need to split it into lines. The whole function body can be simplified to:
```clj
(when (re-find #"INFO:ENGINE: Defold Engine ([^\s]+) \(([^)]+)\)" line)
  line)
```